### DISCUSSION TITLE
您应当如何做才能做到星星之火可以燎原？

### DISCUSSION THREAD

**Discussion by ghost** [WeNeedHome/SummaryOfLoanSuspension#528] [Ideas]
您应当找到一种保存此仓库的方法，实时更新它，并定期告知其他用户备份该仓库的地址。
Git是去中心化的工具，您必须充分利用它，并且必须像Wikileaks一样相互协作。

**Comment by ghost** [comment]
任何隐私泄露行为都是非法的，为保证您的安全，请自觉使用非正式账户进行操作。
Google Chrome及Edge是推荐的浏览器，您应当予以通告。

**Reply by organics2016** [reply]
如果仅仅是浏览器或者账户就能解决问题的话，就不会有呢么多悲剧发生了
这些措施只能保证你的repo安全，不能保证肉身安全。: (

**Reply by ghost** [reply]
我知道单纯的浏览器不能解决问题 但这个repo被盯上是迟早的事 不如借此机会传播信息保护手段 降低被查水表的概率 也是好事

**Reply by organics2016** [reply]
在国内构建一套完整的安全环境需要花费大量的精力，从账号、虚拟机、应用、代理、混淆等多个方面，而且环境本身也是动态的，单从技术上讲。。。所以现在我发现比较好的办法还是肉身出去。
我不赞同在墙内对墙外任何平台产生输入，这样会暴露无疑，虽然我已经这么做了。

**Reply by organics2016** [reply]
我只想表达两句，国内的信息审核让人有点喘不过气。（；；）

**Reply by ghost** [reply]
认同你的回答 这样做风险很大 但在保证一定程度安全的前提下 将方法论传播出去 也是最大化翻墙资源的手段 这也是我翻墙的目的

**Comment by ghost** [comment]
1

**Comment by ghost** [comment]
大家持续顶贴 一起传下去 让更多人看到

**Comment by z-huabao** [comment]
要有自由联络的工具，这样才能团结群众建立组织，不能被随意封号、禁言，不能被任何势力所掌控，包括国家和外部势力，所以它必须是去中心化的，这听着跟区块链很像。

我找到一个类似的mesh组网的APP叫做Tango，可以用来聊天，但是好像没什么人用，有没有专业人士给点建议？

**Reply by NiceAir** [reply]
这明显也是老外

**Comment by ljw-bigtail** [comment]
这是机翻吧

**Comment by NiceAir** [comment]
这货肯定不是中国人，跑来带节奏的，大家擦亮眼睛啊

**Reply by yalo-ddaa** [reply]
那您快回内网去，内网没有老外，没法“带节奏”。

**Comment by mushui-x** [comment]
这特么绝对不是中国人...
